_layouts/tutorial_hands_on.html

```diff
@@ -224,6 +224,58 @@ <h1 id="bibliography">References</h1>
                     {% bibliography --cited %}
                     {% endif %}
 
+                    <h1>Citing this Tutorial</h1>
+                    <p>
+                        <ol>
+                            <li id="citation-text">
+                            {%- include _includes/contributor-author-list-comma.html contributors=page.contributors -%}, {{ page.last_modified_at | date: "%Y" }} <b>{{ page.title }} (Galaxy Training Materials)</b>. <a href="{{ site.baseurl }}{{page.url}}">{{ site.baseurl }}{{page.url}}</a> Online; accessed TODAY
+                            </li>
+                            <li>
+                            Batut et al., 2018 <b>Community-Driven Data Analysis Training for Biology</b> Cell Systems <a href="https://doi.org/10.1016%2Fj.cels.2018.05.012">10.1016/j.cels.2018.05.012</a>
+                            </li>
+                        </ol>
+                    </p>
+
+
+                    <blockquote class="details">
+                      <h3><i class="fa fa-info-circle" aria-hidden="true"></i><span class="visually-hidden">details</span> BibTeX</h3>
+                      <p style="display: none;">
+
+                    <div class="highlighter-rouge"><div class="highlight"><pre class="highlight">
+<code id="citation-code">@misc{% raw %}{{% endraw %}{{topic.name}}-{{page.tutorial_name}},
+    author = "{%- include _includes/contributor-author-list.html contributors=page.contributors -%}",
+    title = "{{ page.title }} (Galaxy Training Materials)",
+    year = "{{ page.last_modified_at | date: "%Y"}}",
+    month = "{{ page.last_modified_at | date: "%m"}}",
+    day = "{{ page.last_modified_at | date: "%d" }}"
+    url = "{% raw %}\url{{% endraw %}{{ site.baseurl }}{{page.url}}{% raw %}}{% endraw %}",
+    note = "[Online; accessed TODAY]"
+}
+@article{Batut_2018,
+        doi = {10.1016/j.cels.2018.05.012},
```

Comment:
Could you use the indentation as above?

Reply:
No, cannot in a `pre` block. It must be left aligned.